LinkerScript.sublime-syntax

```diff
@@ -82,6 +86,17 @@ contexts:
           scope: invalid.illegal.linker
           pop: true
 
+  strings:
```

Comment:
In the GNU linker syntax, double quotes are for surrounding symbol names, so this `strings` section conflicts with the above `symbol_names` section.